src/main/java/org/traccar/protocol/RuptelaProtocolDecoder.java

```diff
@@ -97,10 +97,20 @@ private void decodeParameter(Position position, int id, ByteBuf buf, int length)
             case 2:
             case 3:
             case 4:
+            case 5:
                 position.set("di" + (id - 1), readValue(buf, length, false));
                 break;
-            case 5:
-                position.set(Position.KEY_IGNITION, readValue(buf, length, false) == 1);
+            case 20:
+                position.set("ai3", readValue(buf, length, false));
```

Comment:
What is AI?

Reply:
I was going with the "di" as a digital input and "ai" as an analog input. I guess this touches on the next PREFIX_IN comment to. How do we handle device that has DIN1 and AIN1 as a digital input 1 and analog input 1?

Reply:
"IN" is only for digital inputs. For analog we have "ADC", which is the standard.